Completing the task

stino-x opened the pull request:
i had issues with the former pull request in my linters i hope this is better
it contains a styles h1 element

stino-x committed:
Completing the exercise

i added linters and styled the text Hello microverse!!

stino-x committed:
trying to resolve the linters

my checks weren't performing so i made some changes to try to rectify that in the workflow folder

Nyame-Wolf commented on test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
I suggest you delete this file from github, retain your local copy and add it to .gitignore. This is one of the instructions from the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)
<img width="1670" alt="Screenshot 2023-01-13 at 18 14 09" src="https://user-images.githubusercontent.com/48546992/212354106-a4ce17dc-9917-49b1-b93b-6c93c6afe6db.png">

Nyame-Wolf commented on README.md:
```diff
@@ -1,288 +1,52 @@
 
-After you're finished please remove all the comments and instructions!
--->
+[Live Demo Link](https://livedemo.com)
```
You currently don't have your app live. I suggest you remove this line and retain only the parts of the readme that are relevant for you.

Nyame-Wolf commented on README.md:
```diff
@@ -1,288 +1,52 @@
-## 📝 License <a name="license"></a>
+## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```
Kindly add a licence to your project. Instructions on how to do this are on the note in  line 52 below. You do not need the instructions anymore so once done, kindly remove this.

Nyame-Wolf reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle: 

Hi @stino-x  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

## Highlights :sparkles:

- Linters are passing
- App is working 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

stino-x committed:
editing the read me file

i made the changes stated in the title and i also added test.md to the gitignore file

christianbiring1 reviewed (changes requested):
Hi @stino-x , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
![almost](https://user-images.githubusercontent.com/97100091/212362496-0ab40c26-b51d-4248-8af0-ad7d773e4454.gif)
 

 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 

 ## Required Changes ♻️
- [ ] `Kindly make sure you use a correct README file provided by microverse and personalize it to what applies to the project you just built. YOu can find it` [here](https://github.com/microverseinc/readme-template)
- [ ] `As said in the project requirement, Kindly make sure the test.md file is ignored by git and doesn't appear here on your GitHub`
`Hint: As I can see it already in your gitignore file you may consider deleting it here from github` :smile: 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
stylint check

resolving previous isuues

stino-x committed:
i updated my readme file

i updated my readme file
i also fixed the style lint issue

stino-x committed:
i deleted  the test file

i deleted  the test file as requested

AbayomiOlaoye commented on index.html:
```diff
@@ -0,0 +1,13 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
- [ ] [OPTIONAL] It is a good practice to always include `descriptive title` in our documents/webpage. Good titles on our webpages increase page SEO ranking and also allow screen reader users to access our website with ease. Please consider including `meaningful` title content that tells what your page is about.

AbayomiOlaoye commented on README.md:
```diff
@@ -282,7 +187,3 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [ ] In an attempt to make this ReadMe personalized as the previous reviewer suggested, kindly remove this part of your ReadMe as it is not important to your users. It is more professional to keep files that do not add any benefit to our users. This majorly is a guide that explains what this `License` section is about, and it also directs you on how to use or choose from the many types of licenses that exist.

stino-x replied:
i did remove that 
i think youre looking at the wrong branch maybe

AbayomiOlaoye commented on README.md:
```diff
@@ -282,7 +187,3 @@ I would like to thank...
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
- You have done well by including this path but the path is currently broken - not working 🚯 
- [ ] Please check the link and see how you can make this work. Broken links in our works makes our work unprofessional and suggest that we don't have eyes for details.

AbayomiOlaoye commented on README.md:
```diff
@@ -225,55 +156,29 @@ Example:
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
-
-  - [Answer_1]
+- **[how did you link the two files]**
 
-- **[Question_2]**
+  - [using the css link the the head tag]
```
- [ ] Great questions! 😆 except that you forgot to remove the brace brackets, include question marks and provide answers to your users. Please consider addressing these issues or remove this section completely - It's optional after all 😉

AbayomiOlaoye commented on README.md:
```diff
@@ -225,55 +156,29 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```
- [ ] To make this really customized, I think it is better to remove this generic suggestion provided by the template you used. Remember that users have to writing permission in your code, so there is nothing suggest in that regard. You can however replace with nice crafted statements about your proposed future changes. 🍷

AbayomiOlaoye commented on README.md:
```diff
@@ -225,55 +156,29 @@ Example:
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+no new features will be added
```
- [ ] That's an honest entry there 👍 but how about we pretending that we are going to add some more styling, html content, add nice animations, and dynamism to the site? 😄 This shows that we are conscious of growth and continuous improvement.

AbayomiOlaoye commented on README.md:
```diff
@@ -148,74 +122,31 @@ Example commands:
-## 👥 Authors <a name="authors"></a>
+## 👥 Authors <a name="authors">Iheagwara Augustine</a>
 
 > Mention all of the collaborators of this project.
```
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.

AbayomiOlaoye commented on README.md:
```diff
@@ -111,34 +86,33 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
```
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.

AbayomiOlaoye commented on README.md:
```diff
@@ -111,34 +86,33 @@ After you're finished please remove all the comments and instructions!
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](http://127.0.0.1:5500/index.html))
```
- [ ] This link might only work on your system when your liver server is running because you have the source file opened on your local machine but we don't have access to it. So, since you have not deployed this on any server with global access, I suggest you remove the link, and let people know that the link is not available. What do you think?

AbayomiOlaoye commented on README.md:
```diff
@@ -60,48 +60,23 @@ After you're finished please remove all the comments and instructions!
+
+### Key Features <a name="key-features">hover feature over button</a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
```
- [ ] Kindly show how you are of your achievement by listing what to look out for in this work e.g. `semantic use of html tags` , `Industry standard for best practices` , etc... 🔧

AbayomiOlaoye commented on README.md:
```diff
@@ -60,48 +60,23 @@ After you're finished please remove all the comments and instructions!
 
-### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
```
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.

AbayomiOlaoye commented on README.md:
```diff
@@ -60,48 +60,23 @@ After you're finished please remove all the comments and instructions!
+# 📖 [HELLO-NEW] <a name="about-project">Html/css</a>
 
-> Describe your project in 1 or 2 sentences.
+>  this is the first phase of my first portfolio project
```
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.

AbayomiOlaoye reviewed (changes requested):
Hi @stino-x , 

Great work on making some of the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so_close](https://user-images.githubusercontent.com/99715640/211872359-9a4d36f7-0386-4000-82a4-c274b06a5ce4.gif)


 ## Highlights
- Good PR message ✅ 
- Great commit messages ✅ 
- No linter error ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
read-me update

i added all the changes suggested to me to the best of my ability

stino-x committed:
updated readme

i included a better license link

joskalenda reviewed (approved):
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @stino-x , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
| <img src="https://media2.giphy.com/media/2NpKgBuUZtXpu/giphy.gif"/> |

### Highlights
✅ ... Green linters
✅ ... Good readme design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

joskalenda reviewed (approved):
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @stino-x , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
| <img src="https://media4.giphy.com/media/2RGhmKXcl0ViM/giphy.gif"/> |

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
